tools/appliance/systemvmtemplate/scripts/definition.rb

```diff
@@ -29,7 +29,7 @@
         :os_type_id => 'Debian_64',
         :iso_file => 'debian-9.3.0-amd64-netinst.iso',
         :iso_src => 'https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-9.3.0-amd64-netinst.iso',
-        :iso_sha512 => '8775231d6f56a3d8f116eb64fe048f5cbd2ea0f8c092a1cb7608bcb4106f9c85cb69ce68f53bd381019ab40f1c0316843036daf3fd9107c81c58a240334cc747'
+        :iso_md5 => '8775231d6f56a3d8f116eb64fe048f5cbd2ea0f8c092a1cb7608bcb4106f9c85cb69ce68f53bd381019ab40f1c0316843036daf3fd9107c81c58a240334cc747'
```

Comment:
md5 can't be that lengthy. Is this parameter really checked somewhere since it cannot match? This line is the sha512 signature

Reply:
Thanks for point out, this file is no longer used. I've moved the systemvmtemplate build/export system to be packer based now where configuration is picked from a `template.json` file now. I've removed it now.